kasa/smart/smartdevice.py

```diff
@@ -204,6 +204,14 @@ def _parse_components(components_raw: ComponentsRaw) -> dict[str, int]:
             for comp in components_raw["component_list"]
         }
 
+    def _enable_device_quirks(self) -> None:
```

Comment:
I couldn't make it static to be called in `_parse_components` so I called it just after to fix the `self.components` without touching the `self.components_raw`